### ISSUE TITLE
[bug] panic: integer divide by zero

### ISSUE THREAD

**Issue by abohatyrenko** [centrifugal/centrifugo#667]
Got panic with following content:

```
panic: runtime error: integer divide by zero

goroutine 219 [running]:
github.com/centrifugal/centrifugo/v4/internal/usage.(*Sender).prepareMetrics(0xc000780c00)
	/home/runner/work/centrifugo/centrifugo/internal/usage/usage.go:468 +0x2489
github.com/centrifugal/centrifugo/v4/internal/usage.(*Sender).Start(0xc000780c00, {0x1809be8, 0xc000044030})
	/home/runner/work/centrifugo/centrifugo/internal/usage/usage.go:186 +0x6c9
created by main.main.func1
	/home/runner/work/centrifugo/centrifugo/main.go:617 +0x2e32`
```

...

Centrifugo version is v4.1.4
Client library used is gocent of version 3.2.0
Operating system is Debian 5.10.136-1~deb10u3 (k8s)

...

Deployment has 2 pods

```
    config:
      engine: "redis"
      redis_sentinel_address: "redis-sentinel.svc.cluster.local.:26379"
      redis_sentinel_master_name: "master"

      admin: true

      allowed_origins:
      - "*"

      allow_subscribe_for_client: true

      namespaces:
      - name: "public"
        allow_subscribe_for_client: true`
```

**Comment by FZambia** [comment]
Hello, thanks for the report. Will be fixed in the next release. 

As a workaround you can add `usage_stats_disable: true` to `config` section.

**Comment by FZambia** [comment]
Should be fixed in [v4.1.5](https://github.com/centrifugal/centrifugo/releases/tag/v4.1.5)
